cmd/feed-ingress/main.go

```diff
@@ -165,7 +166,7 @@ func main() {
 		Updaters:                updaters,
 		DefaultAllow:            ingressAllow,
 		DefaultStripPath:        ingressStripPath,
-		DefaultBackendKeepAlive: nginxConfig.KeepaliveSeconds,
+		DefaultBackendKeepAlive: nginxBackendKeepaliveSeconds,
```

Comment:
Pre initialise the controller config struct you mean? Just feels so dirty to me :)